subsys/bluetooth/mesh/Kconfig

```
@@ -1652,18 +1706,27 @@ config BT_MESH_SETTINGS_WORKQ_STACK_SIZE
 	  Size of the settings workqueue stack.
 
 endif # BT_MESH_SETTINGS_WORKQ
-endif # BT_SETTINGS
 
-if BT_MESH_LOG_LEVEL_DBG
+endmenu # Persistent storage
 
-config BT_MESH_DEBUG_USE_ID_ADDR
-	bool "Use identity address for all advertising"
+rsource "shell/Kconfig"
+
+if BT_CONN
+
+config BT_MESH_MAX_CONN
+	int "Maximum number of simultaneous connections used by the stack"
+	default BT_MAX_CONN
+	range 1 BT_MAX_CONN
```

```suggestion
config BT_MESH_MAX_CONN
	int "Maximum number of simultaneous Bluetooth connections"
	default BT_MAX_CONN
	range 1 BT_MAX_CONN
```
Gives a little mor initial info in the menuconfig. The help text should provide the additional info if it is needed.